### DISCUSSION TITLE
模块可能导致zygisk被发现

### DISCUSSION THREAD

**Discussion by MH-Leon** [xfqwdsj/IAmNotADeveloper#34] [Q&A] [answered]
平台：联发科8100
系统：miui14.0.7 安卓13
内核：5.10.136-android12-9
root：kernelSu v0.7.0
magisk模块：shamiko v0.7.4
                      Zygisk next v4-0.81
                      Zygisk-Lsp v1.9.1

检测手段：momo 4.3.1, 云闪付9.3.8

问题现象：在lsp中安装模块但未启用，momo校验通过，云闪付能正常打开允许。

<details>

![Screenshot_2023-11-08-21-48-51-434_com.android.shell.jpg](https://github.com/xfqwdsj/IAmNotADeveloper/assets/60940121/622fa375-aff5-41b9-bb43-88eccf3642b2)

![Screenshot_2023-11-08-21-49-01-223_io.github.vvb2060.mahoshojo.jpg](https://github.com/xfqwdsj/IAmNotADeveloper/assets/60940121/679aaf99-85bc-46b4-8cb5-19be8f1d2f56)

</details>

启用模块，勾选momo和云闪付，momo提示找到zygisk，云闪付闪退无法打开

<details>

![Screenshot_2023-11-08-21-48-28-678_com.android.shell.jpg](https://github.com/xfqwdsj/IAmNotADeveloper/assets/60940121/406fea42-d708-4082-8c78-27704c86163c)

![Screenshot_2023-11-08-21-48-35-821_io.github.vvb2060.mahoshojo.jpg](https://github.com/xfqwdsj/IAmNotADeveloper/assets/60940121/296722f0-6f5d-4efd-89a4-1863c4e4b0b0)

</details>

无论模块中的三个隐藏选项如何配置（我排列组合试过了），甚至全部不打开，只要在lsp中勾选了momo，就会导致zygisk被发现

<details>

![Screenshot_2023-11-08-21-48-18-349_xyz.xfqlittlefan.notdeveloper.jpg](https://github.com/xfqwdsj/IAmNotADeveloper/assets/60940121/1ddf7c31-1690-4fd6-94af-3c69fe57873e)

</details>

**Comment by xfqwdsj** [comment, marked as answer]
这个问题与特定模块无关，只要将应用添加到 LSPosed 作用域中就相当于暴露 Zygisk
